pkgs/development/python-modules/snuggs/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, lib, fetchFromGitHub
+, click, numpy, pyparsing
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "snuggs";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
```

Comment:
Add a comment on why you're using `fetchFromGitHub`.

Reply:
Fixed